app/packages/core/src/plugins/SchemaIO/OperatorIO.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * TODO: FOR TESTING ONLY, REMOVE
+ */
+
+import React from "react";
+import { Box } from "@mui/material";
+import { PluginComponentType, registerComponent } from "@fiftyone/plugins";
+import { SchemaIOComponent } from ".";
+import { types } from "@fiftyone/operators";
+import annotation from "./fixtures/annotation.json";
+import { log, operatorToIOSchema } from "./utils";
+
+registerComponent({
```

Comment:
We could keep this for testing if useful and only register these if dev mode...?

Reply:
Good idea